scripts/schema-drift.ts

```diff
@@ -1,378 +0,0 @@
-const { warn, error } = require("danger")
```

Comment:
@mzikherman - are you sure this file isn't still needed for checking other stuff'? Is it related to our CI message when checking against force at all?

Reply:
I started updating it and got in the weeds and then was trying to think where this is used.

I cannot find the string "schema-drift" anywhere in this repo, or the general artsy code search besides the `package.json` entry (so if something is invoking `yarn schema-drift` that is not checked in anywhere): https://github.com/search?q=org%3Aartsy+schema-drift&type=code

Reply:
Is it invoked from Danger?

Reply:
But would that not show up checked in....somewhere? (via that Artsy org code search)

Reply:
All good!

Reply:
This is pretty old. I think it was ran in peril briefly but something broke and we disabled it. Think Orta might've worked on this originally? Anyway, yeah, probably fine to delete.